firstvoices/backend/search/utils/object_utils.py

```diff
@@ -77,14 +79,25 @@ def hydrate_objects(search_results, request):
 
     # Fetching objects from the database
     dictionary_objects = list(
-        DictionaryEntry.objects.filter(
-            id__in=dictionary_search_results_ids
-        ).prefetch_related(
+        DictionaryEntry.objects.filter(id__in=dictionary_search_results_ids)
```

Comment:
This is great for now. 

A note for our future selves: I logged fw-4928 to move all the eager fetching to the serializers to avoid duplication (e.g., this is slightly different from how the dictionary viewset does it).